src/lib/components/RichText.svelte

```
@@ -0,0 +1,56 @@
+	onMount(() => {
+		void typesetMath();
+	});
+
+	$effect(() => {
+		content;
+		void typesetMath();
+	});
+</script>
+
+<svelte:element this={as} bind:this={element} class={`rich-text ${className}`.trim()}>
+	{@html content ?? ''}
```

ESLint

[error] `{`@html`}` can lead to XSS attack.

(svelte/no-at-html-tags)